src/src/CapabilityCostsWrapper.js

```diff
@@ -1,3 +1,20 @@
+export class CapabilityCosts{
+
+constructor(){
+  this.rechartsCosts = [];
+  this.rawCosts = [];
+  this.average = 0.0;
+  }
+
+  addCost(cost){
+    this.rawCosts.push(cost);
+    this.rechartsCosts.push({name:cost.timestamp, pv:cost.value});
+  }
+   calculateAndSetAverage(){
```

Comment:
This seem unused and empty.
What is the purpose?

Reply:
Unused code, was trying out some stuff but ended up not using it.

I have deleted it